test/CMakeLists.txt

```diff
@@ -321,13 +340,15 @@ if(iodaconv_bufr_ENABLED)
 
     ecbuild_add_test( TARGET  test_iodaconv_bufr_bufrdescription
                       SOURCES bufr/TestBufrDescription.cpp
-                      ARGS ${CMAKE_CURRENT_SOURCE_DIR}/testinput/bufr_test.yaml
+                      ARGS    testinput/bufr_test.yaml
```

Comment:
Can't we use the same YAML here as the other test? If so, remove the redundant YAML file (and remove it from the CMakeLists.txt)

Reply:
yes. done.